src/shared/components/query/GeneSetSelector.tsx

```diff
@@ -94,7 +94,8 @@ export default class GeneSetSelector extends QueryStoreComponent<{}, {}>
 					onChange={option => this.store.geneQuery = option ? option.value : ''}
 				/>
 
-				{!!(this.store.mutSigForSingleStudy.result.length || this.store.gisticForSingleStudy.result.length) && (
+				{/* currently disable the buttons */}
```

Comment:
i'm not sure what this comment means.  currently?   i would just comment out the whole thing

Reply:
and put a comment in explaining that we are hiding these buttons on ___ date